snakefmt/formatter.py

```diff
@@ -85,13 +84,11 @@ def read_black_config(self, path: PathLike) -> black.FileMode:
                 continue
 
             snakecase_config[key] = val
+        if "line_length" not in snakecase_config:
+            snakecase_config["line_length"] = DEFAULT_LINE_LENGTH
```

Comment:
The issue here is we now rely on the global variable `DEFAULT_LINE_LENGTH` in two places. All the default line length inference should be done in the `__init__` I think.

Reply:
Thanks for the feedback, proposing a fix implementing that !